tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -548,11 +548,11 @@ def test_whisper_timestamp_prediction(self):
                     },
                     {
                         "text": " the thousands of spectators, retrievality is not worth thinking about.",
-                        "timestamp": (19.6, 24.98),
+                        "timestamp": (19.6, 26.66),
                     },
                     {
                         "text": " His instant panic was followed by a small, sharp blow high on his chest.",
-                        "timestamp": (24.98, 30.98),
+                        "timestamp": (26.66, 31.060000000000002),
```

Comment:
Can we use something that tests the almost equality and replace with this 31.06? You can use `nested_simplify` on the output at line 529 and this should work.

```suggestion
                        "timestamp": (26.66, 31.06),
```